src/main/java/com/ust/sourcecourse/configuration/repository/ProjectGroupRepository.java

```diff
@@ -9,4 +10,10 @@
 public interface ProjectGroupRepository extends JpaRepository<ProjectGroup, Long> {
 
 	List<ProjectGroup> findByProjectUid(Long uid);
+
+	List<ProjectGroup> findAllByTagsIn(List<String> tags);
```

Comment:
If we are not using these methods, remove it.

Reply:
Resolved